### PR TITLE
add wraped cursor option

### PR DESCRIPTION
**Pull request by pocari**
I have implemented `-cursor-wrap` option.

### CODE REVIEW

File: rplugin/python3/denite/ui/default.py [hunk 1]
```diff
@@ -467,7 +468,10 @@ def move_to_next_line(self):
         elif self.__win_cursor + self.__cursor < self.__candidates_len:
             self.__cursor += 1
         else:
-            return
+            if self.__context['cursor_wrap']:
```
Comment (Shougo) [inline rplugin/python3/denite/ui/default.py]: It should be:

```vim
elif self.__context['cursor_wrap']:
     self.move_to_first_line()
else:
     return

File: rplugin/python3/denite/ui/default.py [hunk 2]
```diff
@@ -476,7 +480,10 @@ def move_to_prev_line(self):
         elif self.__cursor >= 1:
             self.__cursor -= 1
         else:
-            return
+            if self.__context['cursor_wrap']:
```
Comment (Shougo) [inline rplugin/python3/denite/ui/default.py]: Same above.

**Comment by pocari** [comment]
fixed.

**Comment by Shougo** [comment]
LGTM

**Pull request merged**